tilequeue/config.py

```diff
@@ -157,6 +170,7 @@ def default_yml_config():
             'template-path': None,
             'reload-templates': False,
             'formats': ('json',),
+            'layers-to-format': (),
```

Comment:
Doesn't matter here, because we only just iterate through them. It would if we tried to mutate those values anywhere though; we'd get a failure then.

But using lists might be more consistent though since that's how they would look when configured in yaml. I think I just saw the tuple on the line above and followed suit :) I made the updates in 1bab809.